background_scripts/main.coffee

```diff
@@ -116,7 +116,7 @@ TabOperations =
     switch position
       when "start" then tabIndex = 0
       when "before" then tabIndex = request.tab.index
-      when "end" then tabIndex = null
+      when "end" then tabIndex = (if Utils.isFirefox() then 9999 else null)
```

Comment:
Please add a comment explaining why firefox needs this logic and chrome doesn't -- i.e. explain why Firefox has a special case. I'm noticing that the code base has started accumulating firefox-specific logic which works, but isn't well documented as to why we need it, and so it's not going to survive big refactors or changes by relatively novice contributors.